platforms/ios/src/TangramMap/TGMapViewController.h

```diff
@@ -296,8 +302,9 @@ NS_ASSUME_NONNULL_BEGIN
  an API key</a> and load it through your application:
 
  ```swift
- let sceneURL = "https://mapzen.com/carto/walkabout-style-more-labels/walkabout-style-more-labels.yaml";
- view.loadSceneFile(sceneURL, sceneUpdates: [ TGSceneUpdate(path: "sources.mapzen.url_params", value: "{ api_key: \(YOUR_API_KEY) }") ]);
+ let sceneURL = URL("https://mapzen.com/carto/walkabout-style-more-labels/walkabout-style-more-labels.yaml");
+ let sceneUpdates = [ TGSceneUpdate(path: "sources.mapzen.url_params", value: "{ api_key: \(YOUR_API_KEY) }") ];
+ view.loadScene(from: sceneURL, with: sceneUpdates);
```

Comment:
Ok I do not know this at all. But a reference for how objective-c to swift naming transitions work will be good here.